src/utils/AdbWrapper.js

```diff
@@ -484,17 +485,29 @@ export default class AdbWrapper {
     return output.stdout;
   }
 
+
   async getPackageConfig(name) {
     const configPath = `${this.wtfos.packageConfigPath}/${name}/${this.wtfos.packageConfigFile}`;
     const schemaPath = `${this.wtfos.packageConfigPath}/${name}/${this.wtfos.packageConfigSchema}`;
+    const uiSchemaPath = `${this.wtfos.packageConfigPath}/${name}/${this.wtfos.packageConfigUiSchema}`;
 
     try {
       const config = await this.pullJsonFile(configPath);
       const schema = await this.pullJsonFile(schemaPath);
 
+      // gets it's own try as we can tolerate no uiSchema here - we don't want to full fail as if we have no schema/config
+      const uiSchema = await (async() => {
+        try {
+          return await this.pullJsonFile(uiSchemaPath);
+        } catch (e) {
+          return "{}"; // empty uiSchema
```

Comment:
`pullJsonFile` (should?) return a JSON object, why are you returning stringified JSON here?